schema-engine/sql-schema-describer/src/postgres.rs

```diff
@@ -253,7 +253,7 @@ impl PostgresSchemaExt {
         let low = self.exclude_constraints.partition_point(|(id, _)| *id < table_id);
         let high = self.exclude_constraints[low..].partition_point(|(id, _)| *id <= table_id);
 
-        self.exclude_constraints[low..high]
+        self.exclude_constraints[low..low + high]
```

Comment:
This bug is a reminder that maybe a `BTreeMap` would be just fine in the introspection. PSL is a different thing due to it being called a ton everywhere, and we have to optimize. Not the case here.